### PR TITLE
Core/utils/schema validator update

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: core/utils/SchemaValidator.py [hunk 1]
```diff
@@ -15,7 +15,8 @@ def validate(body_response, json_schema):
         try:
             json_schema = JsonFileReader.read(json_path)
             body_json = json.loads(body_response)
-            validate(instance=body_json, schema=json_schema)
-        except:
-            return False
-        return True
+            result = validate(instance=body_json, schema=json_schema)
+            if result is None:
```
Comment (Amet015) [inline core/utils/SchemaValidator.py]: could you use just:
return result is none
whiout IF?

File: trello/jsonfiles/delete_board_schema.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "type": "object",
+  "properties": {
+    "_value": {
+      "type": "null"
+    }
+  },
+  "required": [
+    "_value"
+  ]
+}
```
Comment (Amet015) [inline trello/jsonfiles/delete_board_schema.json]: add new line

File: trello/jsonfiles/delete_card_label_schema.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "type": "object",
+  "properties": {
+    "limits": {
+      "type": "object"
+    }
+  },
+  "required": [
+    "limits"
+  ]
+}
```
Comment (Amet015) [inline trello/jsonfiles/delete_card_label_schema.json]: add new line

File: trello/jsonfiles/delete_organization_schema.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object"
+}
```
Comment (Amet015) [inline trello/jsonfiles/delete_organization_schema.json]: add new line

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

**Pull request merged**